<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>587</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>FileRenamer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <widget class="FrameBuilderList" name="frameBuilderList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
      <widget class="PathTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRename"/>
   <addaction name="actionStop"/>
   <addaction name="actionUndo"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionDarkMode"/>
   <addaction name="actionClearItems"/>
   <addaction name="actionViewLogs"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLogs">
   <property name="windowTitle">
    <string>Logs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="WidgetApplicationLogs" name="dockWidgetContents"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/forward.svg</normaloff>:/res/images/forward.svg</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="toolTip">
    <string>Start renaming (Ctrl+R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/stop.svg</normaloff>:/res/images/stop.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop renaming (Esc)</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/backward.svg</normaloff>:/res/images/backward.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo renaming (Ctrl+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/exit.svg</normaloff>:/res/images/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit application (Ctrl+Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionViewLogs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/res/icons/text-x-preview.ico</normaloff>:/res/icons/text-x-preview.ico</iconset>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
   <property name="toolTip">
    <string>View rename logs (Ctrl+L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionClearItems">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/clear.svg</normaloff>:/res/images/clear.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear file/dir list (Ctrl+K, Ctrl+Del)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K, Ctrl+Del</string>
   </property>
  </action>
  <action name="actionDarkMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/darkmode.svg</normaloff>:/res/images/darkmode.svg</iconset>
   </property>
   <property name="text">
    <string>Dark mode</string>
   </property>
   <property name="toolTip">
    <string>Dark mode (Ctrl+D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PathTableView</class>
   <extends>QTableView</extends>
   <header>path/pathtableview.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetApplicationLogs</class>
   <extends>QWidget</extends>
   <header>widgets/widgetapplicationlogs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrameBuilderList</class>
   <extends>QFrame</extends>
   <header>widgets/framebuilderlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewLogs</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidgetLogs</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetLogs</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionViewLogs</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDarkMode</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionDarkModeTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonSaveSettingsClicked()</slot>
  <slot>onButtonLoadSettingsClicked()</slot>
  <slot>onActionDarkModeTriggered(bool)</slot>
 </slots>
</ui>
